### PR TITLE
Release Notes 1.10

### PR DESCRIPTION
**Pull request by galak**
Some updates to the 1.10 release notes.

### CODE REVIEW

File: doc/release-notes-1.10.rst [hunk 1]
```diff
@@ -35,17 +35,32 @@ Kernel
...
   kernel by passing it garbage.
 
-
 Architectures
 *************
 
 * nrf52: Add support for LOW_POWER state and SYSTEM_OFF
 * Architecture specific memory domain APIs added
+* Added support for the following ARM SoCs:
+  * NXP i.MX RT
```
Comment (dbkinder) [inline doc/release-notes-1.10.rst]: Needs a blank line before this sublist.  (Can you also add a blank line before the sublist on line 25).

File: doc/release-notes-1.10.rst [hunk 2]
```diff
@@ -35,17 +35,32 @@ Kernel
...
+  * Silabs EFM32WG
+  * STM F0
+  * TI MSP432P4xx
 
 Boards
 ******
 
 * Power Management for nrf52 series SOC
+* Added support for the following ARM boards:
+  * 96b_neonkey
```
Comment (dbkinder) [inline doc/release-notes-1.10.rst]: Needs a blank line before this list.

**Review by dbkinder — CHANGES_REQUESTED** [review]
formatting fix needed

File: doc/release-notes-1.10.rst (hunk 1 above)
Comment (MaureenHelm) [reply doc/release-notes-1.10.rst]: This should be `NXP i.MX RT1052` to be more specific (i.MX RT is a family of SoCs)

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by dbkinder — APPROVED** [review]
Guess I'll add the blank line before line 25 on the next pass :)

**Pull request merged**